### ISSUE TITLE
Steep drops in publication counts into 2022

### ISSUE THREAD

**Issue by za158** [georgetown-cset/parat#359]
According to their detail views, every one of the "top ten" PARAT companies according to the default view (https://parat-dev.eto.tech/) posted a decline in AI publications from 2021 to 2022, often an extremely steep decline. This does not seem likely.

Examples
![image](https://github.com/georgetown-cset/parat/assets/58824955/af504fc3-1fcb-42c9-b6aa-8d6d287c371a)
![image](https://github.com/georgetown-cset/parat/assets/58824955/046cda0e-c7d0-43ef-8b89-7824410981e5)
![image](https://github.com/georgetown-cset/parat/assets/58824955/4fe6174f-41e7-4658-8031-21eb5922bb94)

cc @rggelles and (given prior experience) @jamesdunham

**Comment by jmelot** [comment]
I checked a couple of these companies and the big drop was present on 3/6, before my changes to the PARAT pipelines and merged corpus. Presumably this is related to https://github.com/georgetown-cset/cset_article_schema/issues/141 . I'll chat with James about it again when he gets back

**Comment by rggelles** [comment]
Yeah, as @jmelot say, I don't think this is a PARAT-specific issue; it's likely related to the remaining apparent drop in US AI publications that was still outstanding in https://github.com/georgetown-cset/cset_article_schema/issues/141.

**Comment by jmelot** [comment]
I see something similar for google (although there the apparent peak in AI papers is way back in 2020 if you rely only on our affiliations + classifier). Here are the arXiv fulltext counts

<img width="253" alt="Screenshot 2024-06-05 at 8 58 16 AM" src="https://github.com/georgetown-cset/parat/assets/1252409/9ec5c2a2-89f7-4e14-8b89-fc12a0eb25ff">

Adding to the DS sync agenda @rggelles @jamesdunham

**Comment by jmelot** [comment]
Addendum: OA shows an overall publication peak for google in 2021 and a peak for AI (according to a hacky keyword-based definition) in 2020

```
SELECT
  extract(year from publication_date) as year,
  count(distinct(works.id)) as n 
FROM openalex.works
left join unnest(authorships) as authorship
left join unnest(institutions) as institution
where (lower(institution.display_name) like "%google%") or (lower(institution.display_name) like "%alphabet%")
group by year order by year desc
```

<img width="253" alt="Screenshot 2024-06-05 at 9 09 34 AM" src="https://github.com/georgetown-cset/parat/assets/1252409/13d6bdf8-b9d4-42a3-9084-4f1133614ccf">

```
SELECT
  extract(year from publication_date) as year,
  count(distinct(works.id)) as n 
FROM openalex.works
left join unnest(authorships) as authorship
left join unnest(institutions) as institution
where ((lower(institution.display_name) like "%google%") or (lower(institution.display_name) like "%alphabet%")) and (abstract like "%AI%" or lower(abstract) like "%machine learning%" or lower(abstract) like "%artificial intelligence%" or lower(abstract) like "%neural network%" or abstract like "%LLM%")
group by year order by year desc
```

<img width="252" alt="Screenshot 2024-06-05 at 9 15 18 AM" src="https://github.com/georgetown-cset/parat/assets/1252409/6d0e1b80-8d93-441b-8ec2-b0d56d9b9e94">

**Comment by daniel-chou** [comment]
Per our discussion at the the DS Sync on 2024-06-05, @daniel-chou will explore whether more authors jointly publish AI papers (like the trend in High-Energy Physics). One way to start is examining trends of the number of authors per AI paper.

**Comment by rggelles** [comment]
So, I wanted to experiment with a few companies that aren't quite so prolific. Here's Panasonic, using OpenAlex. As you can see, its peak is over 10 years ago.

![image](https://github.com/georgetown-cset/parat/assets/31906945/9d974a7f-fae1-4497-922f-a712dd6f56bb)

And here it is in the full merged corpus:

![image](https://github.com/georgetown-cset/parat/assets/31906945/22cd9c9d-28e3-4e9e-a944-2bb2ddd0f1d2)

Here's just AI. You'll notice the peak is now much later, either 2019 or 2022 depending on how you interpret:

![image](https://github.com/georgetown-cset/parat/assets/31906945/fee556be-7435-472b-a9f5-d22f68b29a2b)

I've excluded arXiv metadata because it barely appears -- just 3 total mentions. But here's arXiv fulltext:

![image](https://github.com/georgetown-cset/parat/assets/31906945/af8e52d4-b60a-40c3-92c7-c3e74813df99)

In this one we get a 2023 peak! So the pattern does persist with the somewhat less popular company.

I'll try one more, even less famous, below, and then do some more experiments.

**Comment by rggelles** [comment]
Okay, now let's try with Schlumberger.

For OA, we're seeing a total paper peak all the way back in 2015:

![image](https://github.com/georgetown-cset/parat/assets/31906945/430b9810-6e2b-483f-9762-d888894a1567)

For the full merged corpus, it's in 2015 as well:

![image](https://github.com/georgetown-cset/parat/assets/31906945/1e2ab704-6791-44e8-9923-9f9d7c90d660)

Focusing on just AI, we see it later, in 2022:

![image](https://github.com/georgetown-cset/parat/assets/31906945/abd2c544-29c0-4112-9f5c-925594bd5210)

Archive metadata, again, has nearly no results to speak of, and what's there is largely from the late 1990s. Archive fulltext, however, has a peak in 2019:

![image](https://github.com/georgetown-cset/parat/assets/31906945/4d0db772-b566-4904-9331-ee56554884be)

I'm not sure what this says. Even Panasonic runs the risk of being a "relevant" enough company to be mentioned in arXiv papers for reasons other than affiliations. Schlumberger probably doesn't. So this peak is probably natural, and it's much earlier than the other ones we've seen. I think my next step is I'm going to take a look at the 2023 paper fulltext and see where the actual appearances of the affilations are showing up.

**Comment by rggelles** [comment]
When I run the following instead of the original arXiv fulltext query, I get numbers that are closer to accurate:

```
select
  extract(year from created) as year,
  count(distinct(id)) as n
from 
  gcp_cset_arxiv_metadata.arxiv_metadata_latest
left join
  gcp_cset_arxiv_full_text.fulltext
using(id)
where lower(SUBSTRING(original_text, 0, 300)) like "%panasonic%"
group by year
order by year desc
```

![image](https://github.com/georgetown-cset/parat/assets/31906945/574a7d7f-5ed4-4f66-bfc4-b016608e646f)

They're not perfect, because in some cases affiliations are footnoted and show up at the bottom of the page instead of near the top, but they're quite close.

Trying this with alibaba, we get:

![image](https://github.com/georgetown-cset/parat/assets/31906945/a5694086-5c49-4c67-a0f9-bb0bb3cc15bf)

So we're still getting a 2023 peak, as with Panasonic. But it's not a particularly significant one. And counts actually go down (slightly) from 2021 to 2022.

What about google?

![image](https://github.com/georgetown-cset/parat/assets/31906945/4d48dcac-ff7e-48ef-b0aa-0fb832e1a22d)

Again, we do see a peak in 2023! But before that we see a noticeable drop in 2021 and 2022, before finally returning to counts only barely above 2020 in 2023. So we're just not seeing consistency here, which isn't surprising given that we're looking at individual companies, across all their research (not just AI!) over a volatile period of time, in only one research source that has a very particular use.

Basically: I'm just not sure the arXiv numbers are meaningful for telling us about anything other than trends in arXiv.

**Comment by jmelot** [comment]
Thanks for doing this analysis (and I admire your annotation stamina...). Checking the first segment of the FT for mentions was a good idea. I also annotated a few papers for alibaba and noticed that there were a bunch of false positives (mentions in references, research results). When I switched to an email address-based search 10/10 I checked had alibaba in the affiliations list

<img width="399" alt="Screenshot 2024-06-06 at 12 20 45 PM" src="https://github.com/georgetown-cset/parat/assets/1252409/a155293d-ad49-4c34-b7b9-add0cb53de25">

I tried the same thing with google and 5/5 of these I checked had google affiliations. If anything, these may be undercounts, but they should be much higher-precision

<img width="381" alt="Screenshot 2024-06-06 at 12 32 56 PM" src="https://github.com/georgetown-cset/parat/assets/1252409/b1eb8a05-631f-4d3f-928d-ca3096c453c7">

Anyway, as you say, these are counts of all papers in arXiv. We could easily either search the fulltext for key terms, mentions of cs.AI, or join to the predictions table to filter to AI though. I think that's still worth doing

**Comment by rggelles** [comment]
Okay, I took a look arXiv fulltext AI, using your email method and joining to the predictions table.

Here's Alibaba:

![image](https://github.com/georgetown-cset/parat/assets/31906945/7263ac29-98a3-47be-9b03-78a75642852e)

and Google:

![image](https://github.com/georgetown-cset/parat/assets/31906945/7eeea929-fad0-4234-b33a-537af7bdd6b5)

I tried Panasonic with this method and got zero papers; from a quick scan of the ones in the previous list, my guess is this is some combo of a lot of Panasonic's arXiv papers not being AI and a lot of Panasonic's authors not including their emails for whatever reason.

Still, this is interesting: Alibaba still has a peak in 2023, but Google doesn't (which is also true in your query above, even when we don't narrow to AI-only.) Instead, it peaks in 2020.

**Comment by jmelot** [comment]
Hmmmm that is very interesting! I'll try this with a couple more of the top few companies in parat after the DT meeting...

**Comment by rggelles** [comment]
I messed up the Panasonic query (my fault for rushing; had a delivery person I was dealing with.

So that's here. Back to a 2023 peak, at...3 papers more than 2022:

![image](https://github.com/georgetown-cset/parat/assets/31906945/8b40f9db-4587-43b1-8749-fbc65cfdee2f)

And here's Nvidia, which has a more apparent peak in 2023:

![image](https://github.com/georgetown-cset/parat/assets/31906945/fc7bb06a-e6a7-4bbf-abc1-0a97ae880325)

Bosch, also a peak in 2023, but with very small counts, despite an apparently high number of total AI papers?:

![image](https://github.com/georgetown-cset/parat/assets/31906945/9f582d3d-d87b-4fe1-a41c-0ba5444c8e0e)

Okay, I think my next step is to look at what percentage of AI papers are on arXiv by year. @jmelot Do you know which of our sources incorporate arXiv in their ingest? I'd ideally also like to look at what percentage of AI papers are solely on arXiv by year, but if most of our sources include all of arXiv in them anyway that will be rather difficult.

**Comment by rggelles** [comment]
I wanted to take a look at these percentages for some datasets that aren't arXiv, to see how they fluctuate year-to-year and if that could be affecting things.

Here's WOS overall:

![image](https://github.com/georgetown-cset/parat/assets/31906945/283cec77-ff8d-4ee3-8857-d42d8cf98069)

And WOS AI-only:

![image](https://github.com/georgetown-cset/parat/assets/31906945/13b327b6-e4bf-4d60-bf2e-c944828f84cb)

You'll notice in both cases a pretty steep drop in the percentage of our corpus that comes from WOS in 2023 vs. in 2021 and 2022. In the first case, the 2021 and 2022 numbers are big rises from 2020 and before, but for AI, they seem pretty standard, so the 2023 drop is a major outlier. This might be one reason our 2023 numbers look so different; maybe we're relying on WOS in some way (e.g. maybe they do better at filling in affiliations?) and their data takes longer to fill in for recent years?

Following up on this, let's look at the percentage of papers that have affiliations in WOS by year:

![image](https://github.com/georgetown-cset/parat/assets/31906945/986e6540-0769-44a4-b4eb-8493dc0d0f63)

Let's compare this to the total merged corpus:

![image](https://github.com/georgetown-cset/parat/assets/31906945/7cf749e6-8ae1-43f6-8154-575ee2ca2c7a)

Okay, yes, WOS has much much higher affiliation percentages than the total merged corpus. So a steep drop in the percentage of our dataset coming from WOS in 2023 would make a big difference in how many affiliations we're getting for that year.

**Comment by jmelot** [comment]
First, _thank you_ for looking into this! I really appreciate the help tracking this down.

I think you're onto something with the WOS. They are mostly providing us with 2024 papers - in fact more so than say OA over a roughly equivalent time interval

<img width="856" alt="Screenshot 2024-06-10 at 3 58 06 PM" src="https://github.com/georgetown-cset/parat/assets/1252409/aac8cfe8-1d38-485b-b823-5d1caedcfd1b">
<img width="659" alt="Screenshot 2024-06-10 at 3 57 57 PM" src="https://github.com/georgetown-cset/parat/assets/1252409/2d1bbf83-00b2-4b45-9e59-2af35d7df148">

But WOS is also providing a higher fraction of 2023 papers still

<img width="651" alt="Screenshot 2024-06-10 at 4 01 16 PM" src="https://github.com/georgetown-cset/parat/assets/1252409/e2f6d31d-03b7-4937-b82b-7eda696b2094">
<img width="846" alt="Screenshot 2024-06-10 at 4 00 56 PM" src="https://github.com/georgetown-cset/parat/assets/1252409/ed54b462-7611-459b-8569-74eab8016e18">

"The WOS process is slower but results in more affiliations" seems like a reasonable guess about what's happening. If no one has anything further they want to look at here, we can push the interval we mark incomplete back a year in parat and anywhere else we're displaying graphs based on affiliations.

**Comment by daniel-chou** [comment]
Showing prelim results with Alibaba AI output (paper is analysis unit) that peaked in 2021:

<img width="390" alt="Screenshot 2024-06-12 at 12 01 41" src="https://github.com/georgetown-cset/parat/assets/46931508/cd371b23-86e3-49f2-8f68-2f3b53837a32">

```
WITH
  ai_papers AS (
  SELECT
    DISTINCT merged_id
  FROM
    article_classification.predictions
  WHERE
    ai_filtered),
  alibaba_papers AS (
  SELECT
    DISTINCT merged_id
  FROM
    literature.affiliations
  WHERE
    LOWER(org_name) LIKE "%alibaba%"
    OR org_name LIKE "%阿里巴巴%"
    OR LOWER(org_name) LIKE "%alicloud%"
    OR org_name LIKE "%阿里云%"
    OR org_name LIKE "%DAMO%"
    OR org_name LIKE "%达摩%" )
SELECT
  year,
  COUNT(DISTINCT merged_id) AS ai_papers
FROM
  literature.papers
INNER JOIN
  ai_papers
USING
  (merged_id)
INNER JOIN
  alibaba_papers
USING
  (merged_id)
WHERE
  year > 2010
  AND year < EXTRACT(YEAR
  FROM
    CURRENT_DATE())
  AND (doctype = "Journal"
    OR doctype = "Conference"
    OR doctype = "Preprint")
GROUP BY
  year
ORDER BY
  year DESC
```

**Comment by daniel-chou** [comment]
Author-based prelim results with Alibaba AI output (author-paper is analysis unit) also peaked in 2021:

<img width="395" alt="Screenshot 2024-06-12 at 12 01 28" src="https://github.com/georgetown-cset/parat/assets/46931508/1d235a4f-03c8-464c-9738-905edd85b05d">

Note the standardization cleanup of author names.

```
WITH
  ai_papers AS (
  SELECT
    DISTINCT merged_id
  FROM
    article_classification.predictions
  WHERE
    ai_filtered),
  alibaba_papers AS (
  SELECT
    DISTINCT merged_id
  FROM
    literature.affiliations
  WHERE
    LOWER(org_name) LIKE "%alibaba%"
    OR org_name LIKE "%阿里巴巴%"
    OR LOWER(org_name) LIKE "%alicloud%"
    OR org_name LIKE "%阿里云%"
    OR org_name LIKE "%DAMO%"
    OR org_name LIKE "%达摩%" ),
  alibaba_authors_ai AS (
  SELECT
    DISTINCT year,
    merged_id,
    -- In Merged Corpus, the Chinese name "Wang, Hao" appears as "Wang, Hao", "Hao Wang", and "Hao WANG" for the same paper, so we need to deduplicate
    -- Extract string before the first comma as the surname and after the first comma as the given name
    -- TODO: suffixes and variants with and without with middle initials in the same record
    INITCAP(LOWER(REPLACE(TRIM(CONCAT(TRIM(COALESCE(REGEXP_EXTRACT(author_name, "(.+?), "), author_name)), " ", TRIM(COALESCE(REGEXP_EXTRACT(author_name, ".+?, (.*)"), "")))), "  ", " "))) AS author_name
  FROM
    literature.papers
  INNER JOIN
    literature.authors
  USING
    (merged_id)
  INNER JOIN
    ai_papers
  USING
    (merged_id)
  INNER JOIN
    alibaba_papers
  USING
    (merged_id)
  WHERE
    year > 2010
    AND year < EXTRACT(YEAR
    FROM
      CURRENT_DATE())
    AND (doctype = "Journal"
      OR doctype = "Conference"
      OR doctype = "Preprint") )
SELECT
  -- DON'T use DISTINCT becauase we're counting author-paper and not just paper
  year,
  COUNT(author_name) AS ai_author_papers
FROM
  alibaba_authors_ai
GROUP BY
  year
ORDER BY
  year DESC
```

**Comment by jmelot** [comment]
Thanks again everyone. I'm going to open a PR pushing the incomplete data interval back a year in parat closing this issue. We can do a little "read out" in the next DS sync as well

**Comment by jamesdunham** [comment]
I can contribute a few more results here, which suggest to me we're looking at an issue with org affiliation countries.

https://docs.google.com/spreadsheets/d/1WNTkXizJ575trk0ek-9DW-fxpKM5Vawg_ZK8jd8blDc/edit?gid=665584497#gid=665584497

**Comment by jmelot** [comment]
Closing after discussion in DS sync
